Subject: [PATCH] Use absolute imports
---
Index: WallProjections/Scripts/hotspot_detection.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WallProjections/Scripts/hotspot_detection.py b/WallProjections/Scripts/hotspot_detection.py
--- a/WallProjections/Scripts/hotspot_detection.py	
+++ b/WallProjections/Scripts/hotspot_detection.py	
@@ -3,20 +3,14 @@
 import cv2
 import mediapipe as mp
 
-# noinspection PyPackages
-from .Helper.EventHandler import EventHandler
-# noinspection PyPackages
-from .Helper.Hotspot import Hotspot
-# noinspection PyPackages
-from .Helper.VideoCapture import VideoCapture
-# noinspection PyPackages
-from .Helper.logger import setup_logger
-# noinspection PyPackages
-from .Interop import numpy_dotnet_converters as npnet
-# noinspection PyPackages
-from .Interop.json_dict_converters import json_to_3dict
-# noinspection PyPackages
-from .calibration import Calibrator
+# TODO fix import issues with Daniel
+from Helper.EventHandler import EventHandler
+from Helper.Hotspot import Hotspot
+from Helper.VideoCapture import VideoCapture
+from Helper.logger import setup_logger
+from Interop import numpy_dotnet_converters as npnet
+from Interop.json_dict_converters import json_to_3dict
+from calibration import Calibrator
 
 logger = setup_logger("hotspot_detection")
 
Index: WallProjections/Scripts/calibration.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WallProjections/Scripts/calibration.py b/WallProjections/Scripts/calibration.py
--- a/WallProjections/Scripts/calibration.py	
+++ b/WallProjections/Scripts/calibration.py	
@@ -1,13 +1,8 @@
 import logging
-import datetime
-# noinspection PyPackages
-from .Helper.Calibrator import Calibrator
-# noinspection PyPackages
-from .Interop import numpy_dotnet_converters as npnet
-# noinspection PyPackages
-from .Interop.json_dict_converters import json_to_2dict
-# noinspection PyPackages
-from .Helper.logger import setup_logger
+from Helper.Calibrator import Calibrator
+from Interop import numpy_dotnet_converters as npnet
+from Interop.json_dict_converters import json_to_2dict
+from Helper.logger import setup_logger
 
 logger = setup_logger("calibration")
 
Index: WallProjections/Scripts/Helper/Calibrator.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/WallProjections/Scripts/Helper/Calibrator.py b/WallProjections/Scripts/Helper/Calibrator.py
--- a/WallProjections/Scripts/Helper/Calibrator.py	
+++ b/WallProjections/Scripts/Helper/Calibrator.py	
@@ -2,10 +2,8 @@
 import numpy as np
 from cv2 import aruco
 
-# noinspection PyPackages
-from .VideoCapture import VideoCapture
-# noinspection PyPackages
-from .logger import get_logger
+from Helper.VideoCapture import VideoCapture
+from Helper.logger import get_logger
 
 
 DICT = aruco.getPredefinedDictionary(aruco.DICT_7X7_100)
diff --git a/WallProjections/Scripts/Helper/__init__.py b/WallProjections/Scripts/Helper/__init__.py
new file mode 100644
diff --git a/WallProjections/Scripts/Internal/__init__.py b/WallProjections/Scripts/Internal/__init__.py
new file mode 100644
diff --git a/WallProjections/Scripts/Interop/__init__.py b/WallProjections/Scripts/Interop/__init__.py
new file mode 100644
